### PR TITLE
Pass package.json version to Quiltt platform app to track sdk version

### PR DESCRIPTION
**Pull request by tom-quiltt**
Extract `package.json` version and modify `version.ts` in build time, pass the version to platform app to consume.
I don't a way to track core package version.

### CODE REVIEW

File: ECMAScript/react-native/src/components/QuilttConnector.tsx [hunk 1]
```diff
@@ -33,7 +34,7 @@ export const QuilttConnector = ({
   const webViewRef = useRef<WebView>(null)
   const { session } = useQuilttSession()
   oauthRedirectUrl = encodeURIComponent(oauthRedirectUrl)
-  const connectorUrl = `https://${connectorId}.quiltt.app/?mode=webview&oauth_redirect_url=${oauthRedirectUrl}&sdk=react-native` // @todo append version from package.json
+  const connectorUrl = `https://${connectorId}.quiltt.app/?mode=webview&oauth_redirect_url=${oauthRedirectUrl}&sdk=react-native&version=${version}`
```
Comment (rubendinho) [inline ECMAScript/react-native/src/components/QuilttConnector.tsx:37]: To better differentiate from the API version, can we rename this `sdk_version`?

**Review by rubendinho — APPROVED** [review]
Very neat. 

Can we do the same for React SDK?

File: ECMAScript/react-native/src/components/QuilttConnector.tsx (hunk 1 above)
Comment (tom-quiltt) [reply ECMAScript/react-native/src/components/QuilttConnector.tsx:37]:
```suggestion
  const connectorUrl = `https://${connectorId}.quiltt.app/?mode=webview&oauth_redirect_url=${oauthRedirectUrl}&sdk=react-native& sdk_version =${version}`
```

Comment (rubendinho) [reply ECMAScript/react-native/src/components/QuilttConnector.tsx:37]: yeah i think this works

Comment (tom-quiltt) [reply ECMAScript/react-native/src/components/QuilttConnector.tsx]: Switched to `agent=react-native-${version}` based on @sirwolfgang's input.
https://linear.app/quiltt/issue/Q-679/track-version-in-react-and-core-sdk#comment-add19507

**Pull request merged**